content/drone-plugins/drone-ansible/index.md

```diff
@@ -0,0 +1,111 @@
+---
+version: '1.0'
+date: 2019-02-15T00:00:00+00:00
+title: Ansible
+author: tboerger
+tags: [ infrastructure, ops, ansbile, deploy  ]
+logo: ansible.svg
+repo: drone-plugins/drone-ansible
+image: plugins/ansible
+---
+
+The Ansible plugin can be used to run ansible plays. The below sample pipeline configuration demonstrates simple usage:
+
+```yaml
+kind: pipeline
+name: default
+
+steps:
+- name: check ansible syntax
+  image: plugins/ansible:1
+  environment:
+    additional_var:
+      from_secret: additional_var
+    another_var: foo
+  settings:
+    playbook: ansible/playbook.yml
+    galaxy: ansible/requirements.yml
+    inventory: ansible/inventory
+    syntax_check: true
+  when:
+    event:
+    - pull_request
+
+- name: apply ansible playbook
+  image: plugins/ansible:1
+  environment:
+    additional_var:
+      from_secret: additional_var
+    another_var: foo
+  settings:
+    playbook: ansible/playbook.yml
+    galaxy: ansible/requirements.yml
+    inventory: ansible/inventory
+    private_key:
+      from_secret: ansible_private_key
+    vault_password:
+      from_secret: ansible_vault_password
+  when:
+    event:
+    - push
+    - tag
+
+---
+kind: secret
+
+data:
+  ansible_private_key: ANSIBLE_PRIVATE_KEY
+  ansible_vault_password: ANSIBLE_VAULT_PASSWORD
+```
+
+
+# Parameter Reference
+
+requirements: path to python requirements
```

Comment:
Please use the same notation as the other docs, this will be rendered differently in markdown.

Reply:
fixed